optimum/exporters/openvino/__main__.py

```diff
@@ -332,6 +332,36 @@ class StoreAttr(object):
                 return model
 
             GPTQQuantizer.post_init_model = post_init_model
+    elif library_name == "diffusers" and is_openvino_version(">=", "2024.6"):
+        if Path(model_name_or_path).is_dir():
+            path = Path(model_name_or_path)
+        else:
+            from diffusers import DiffusionPipeline
+            path = DiffusionPipeline.download(model_name_or_path)
```

Comment:
in general case, it should be also extended with pipeline loading args from main_export (trust_remote_code, revision, subfolder and so on) that may affect downloading process